### ISSUE TITLE
Malware?

### ISSUE THREAD

**Issue by WinFan1234** [Endermanch/XPKeygen#26]
When I execute this it appears this: 
Malware file

This application may harm your device. It is possible that your personal confidential information may also be at risk.

Application: XPKeygen (2).exe

Publisher: Unknown publisher

Does it have malware, Do I run it?

**Comment by thepwrtank18** [comment]
Any executable file that doesn't have a certificate (which costs ~$350 a year) will give the "unknown publisher, are you sure" warning.

**Comment by techguy16** [comment]
@WinFan1234 No, it doesn't have malware. Press "More info" and "Run anyway" (or at least that's what you did when I last used Windows 12 months ago)

**Comment by figsgksss** [comment]
its a virus i when i runned it shut off my pc and and i cant turn it on

**Comment by thepwrtank18** [comment]
please provide proof of this happening

**Comment by techguy16** [comment]
> its a virus i when i runned it shut off my pc and and i cant turn it on

It's likely a hardware issue, not to do with this program. If you believe this is the case, show proof.

**Comment by DEMnetwork** [comment]
But MalwereBytes Quarentine The Program

**Comment by DEMnetwork** [comment]
And Google Block the Download

**Comment by thepwrtank18** [comment]
Yes, because false detections exist.

**Comment by thepwrtank18** [comment]
Yes, because false detections exist.

**Comment by techguy16** [comment]
Obviously, Antiviruses would do that because they're designed for people who click on every link in the entire world. They're designed to make people scared and make up reasons to delete the file.

**Comment by Luke-Software1024** [comment]
For me, it says:
Couldn't download - virus detected
XPKeygen.exe was blocked as unsafe by Microsoft Defender SmartScreen.

**Comment by thepwrtank18** [comment]
Turn off Windows Defender *temporarily*, download it, then whitelist it. Then, turn it back on.

**Comment by Luke-Software1024** [comment]
> its a virus i when i runned it shut off my pc and and i cant turn it on

Laptop?

**Comment by DEMnetwork** [comment]
> For me, it says: Couldn't download - virus detected XPKeygen.exe was blocked as unsafe by Microsoft Defender SmartScreen.

I Installed Windows 11 5 Months Ago And I Don't Enable SmartScreen

**Comment by DEMnetwork** [comment]
But I Could Execute XPKeyGen.exe And No Malwere Activity

**Comment by Luke-Software1024** [comment]
> > For me, it says: Couldn't download - virus detected XPKeygen.exe was blocked as unsafe by Microsoft Defender SmartScreen.
> 
> I Installed Windows 11 5 Months Ago And I Don't Enable SmartScreen

Then how do you disable it?

**Comment by Luke-Software1024** [comment]
> But I Could Execute XPKeyGen.exe And No Malwere Activity

But what about "Couldn't download - virus detected"? "XPKeygen.exe was blocked as unsafe by Microsoft Defender SmartScreen." was MDS, and I just disabled it.

**Comment by frenchtrainspotter** [comment]
If you are not sure, you can execute it on a VM

**Comment by Henrisen** [comment]
Its not malware

**Comment by frenchtrainspotter** [comment]
> Its not malware

I think at this point it's not needed anymore to say it. Just let the 2 guys that don't believe it, it's not by saying it again that their mind will change.

Also, I think the thread should be closed and the issue as resolved

**Comment by omega552003** [comment]
> > Its not malware
> 
> I think at this point it's not needed anymore to say it. Just let the 2 guys that don't belive it, it's not by saying it again that their man will change.
> 
> Also I think the thread should be closed and the issue as resolved

Technically yes, but it probably should stay open as I got a false positive with Firefox. This should be a "Known/Accepted Issue"

**Comment by PleaseTeachMeLUA** [comment]
> please provide proof of this happening

Ik this comment was posted in december, so don't criticize me for it. But I just made this account to show the proof that nobody showed.
![Screenshot 2024-01-25 222531](https://github.com/Endermanch/XPKeygen/assets/157775174/a2b8d8be-9b6b-4a26-94a8-41254e14e3c0)
![Screenshot 2024-01-25 222556](https://github.com/Endermanch/XPKeygen/assets/157775174/af36b809-23c1-49b4-b86d-2289dd73f051)

**Comment by frenchtrainspotter** [comment]
> > please provide proof of this happening
> 
> Ik this comment was posted in December, so don't criticize me for it. But I just made this account to show the proof that nobody showed. ![Screenshot 2024-01-25 222531](https://private-user-images.githubusercontent.com/157775174/304163706-a2b8d8be-9b6b-4a26-94a8-41254e14e3c0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3NjIxMTAsIm5iZiI6MTcwNzc2MTgxMCwicGF0aCI6Ii8xNTc3NzUxNzQvMzA0MTYzNzA2LWEyYjhkOGJlLTliNmItNGEyNi05NGE4LTQxMjU0ZTE0ZTNjMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMlQxODE2NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2E1MDk4NTA3MGQ5YzhlNDIxYzM4ZTJlOTYwNTY0NDA0MzBmMWVkOGQ5ZTlmMGI5ZWYzNDkyYWU3NmY1NWQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2OOD0X7tEI_C9L2chPOjaJHxY3lK91sglNcRwRv1Agw) ![Screenshot 2024-01-25 222556](https://private-user-images.githubusercontent.com/157775174/304163765-af36b809-23c1-49b4-b86d-2289dd73f051.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3NjIxMTAsIm5iZiI6MTcwNzc2MTgxMCwicGF0aCI6Ii8xNTc3NzUxNzQvMzA0MTYzNzY1LWFmMzZiODA5LTIzYzEtNDliNC1iODZkLTIyODlkZDczZjA1MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMlQxODE2NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MjM4NWNkYTQzM2M4ZWI3NDNkZjQyNGEyN2MxMGU5NDE1N2M4ZjVhMzU5ZGRjY2ExNzliNTJkMTYwNjk5ODU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.39TxLkPWKdmjSwIxlZFZvA0B7Uj1ct8502ZlCMoL91U)

1) It was about someone claiming it did shutdown their PC and they can't boot up anymore
2) It's a false positive

I still respect you for doing something else than saying "yes it is " "no its not"

**Comment by rock3tsprocket** [comment]
no

**Comment by jaavierdz** [comment]
what the fuck, what a retard conversation. Obviousely it is  **NOT A VIRUS**

**Comment by techguy16** [comment]
I agree, people need to know that it isn't a virus and obviously antiviruses will pick it up as something.

The following is a list of issues of people reporting malware, mostly related to the previous sentence:
- #26
- #25
- #23
- #20
- #18
- #17
- #13
- #9
- #6
